src/data/icons.json

```diff
@@ -54,6 +55,7 @@
   { "name": "AnalyticsReport", "unicode": "F1E1" },
   { "name": "AnchorLock", "unicode": "F511" },
   { "name": "Annotation", "unicode": "E924" },
+  { "name": "ApacheIvyLogo32", "unicode": "F5D1" },
```

Comment:
Think you've called this icon out in your PR comment, but just want to be sure we remove it if it's a third-party icon.